var/spack/repos/builtin/packages/chombo/package.py

```diff
@@ -0,0 +1,130 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+from shutil import copyfile
+import glob
+
+
+class Chombo(MakefilePackage):
+    """The Chombo package provides a set of tools for implementing finite
+       difference and finite-volume methods for the solution of partial
+       differential equations on block-structured adaptively refined
+       logically rectangular (i.e. Cartesian) grids. Note: Chombo download
+       requires human registration process."""
+
+    homepage = "https://anag-repo.lbl.gov/svn/Chombo/release"
+    url      = "https://anag-repo.lbl.gov/svn/Chombo/release/3.2"
+
+    version('3.2', svn='https://anag-repo.lbl.gov/svn/Chombo/release/3.2')
+
+    variant('mpi', default=True, description='Enable MPI parallel support')
+    variant('dims', default=3, description='Number of PDE dimensions [1-6]')
+    variant('hdf5', default=True, description='Enable HDF5 support')
+
+    patch('hdf5-16api.patch', when='@3.2', level=0)
+    patch('Make.defs.local.template.patch', when='@3.2', level=0)
+
+    depends_on('lapack')
```

Comment:
> Isn't that taken care of because lapack itself depends on blas?

I am not sure, it does depend indeed, but theoretically you may have a package which provided both lapack and blas (e.g. `atlas`), but when you write `depends_on('lapack')`, i think `blas` will not be a part of the dependency as `lapack` virtual requirement will be fulfilled by a single package, i.e. `atlas`. Long story short -- i think it's safer to write both.